src/App.tsx

```diff
@@ -12,155 +15,42 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={({ isActive }) =>
+              classNames('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              })
+            }
+            to="/"
+          >
             Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          </NavLink>
+
+          <NavLink
+            className={({ isActive }) =>
+              classNames('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              })
+            }
+            to="/people"
           >
             People
-          </a>
+          </NavLink>
```

Comment:
Those links could be separate component or create constatnt object with `NAV_ITEMS` and map throught them, to not repeat yourself.